client/source/js/modules/model/calibration-ctrl.js

```diff
@@ -7,6 +7,11 @@ define(['./module', 'angular', 'underscore'], function (module, angular, _) {
     $scope.projectInfo = info;
     $scope.canDoFitting = $scope.projectInfo.can_calibrate;
     $scope.needData = !$scope.projectInfo.has_data;
+    $scope.$on('onAfterRender', function (e){ $scope.onAfterGraphRender() });
+    $scope.tallestGraphHeight = 0;
+    $scope.renderedGraphs = 0; // counts how many Graphs are rendered (see onAfterGraphRender for more on this)
+
+    angular.kk = $scope;
```

Comment:
What you mean still?
That's the core of the solution.

Ah you mean the `$angular.kk`  LOL no that's to remove, good catch :+1: